tests/common.go

```diff
@@ -317,7 +317,7 @@ func CollectSupport() {
 			expect(nodes).NotTo(beEmpty())
 
 			journalCmd := fmt.Sprintf(
-				"echo t > /proc/sysrq-trigger && journalctl -l > ~/all_journal_%v",
+				"journalctl -l > ~/all_journal_%v",
```

Comment:
remove the journal too.. not needed.

Reply:
@harsh-px If we remove journalctl command as well there is no need for this function at all then, should we just disable this func?

Reply:
Removed CollectSupport()